numba/tests/test_parfors.py

```diff
@@ -1174,6 +1174,20 @@ def test_impl(n):
         # as a different name
         self.assertEqual(countArrayAllocs(test_impl, (types.intp,)), 1)
 
+    @skip_unsupported
+    def test_reduction_var_reuse(self):
```

Comment:
Once array reduction support is merge I think a test similar to this but with `acc` as an array would be beneficial. Thanks.

Reply:
Done.

Reply:
Done.

Reply:
Thanks.